apps/shared/avifjpeg.c

```diff
@@ -366,7 +366,10 @@ avifBool avifJPEGRead(const char * inputFilename,
         rgb.format = AVIF_RGB_FORMAT_RGB;
         rgb.chromaDownsampling = chromaDownsampling;
         rgb.depth = 8;
-        avifRGBImageAllocatePixels(&rgb);
+        if (avifRGBImageAllocatePixels(&rgb) != AVIF_RESULT_OK) {
+            fprintf(stderr, "Conversion to YUV failed: %s (out of memory)\n", inputFilename);
```

Comment:
Optional: use avifResultToString(return value) in the error message, rather than "out of memory". You do this in some files.

If you make this change, make the same change to other similar error messages.